tests/sles4sap/hana_cluster.pm

```diff
@@ -78,9 +97,8 @@ sub run {
         sleep bmwqemu::scale_timeout(30);
         $self->do_hana_sr_register(node => $node1);
         sleep bmwqemu::scale_timeout(10);
-        my $start_cmd = "su - $sapadm -c 'sapcontrol -nr $instance_id -function StartSystem HDB'";
-        assert_script_run $start_cmd;
-        my $looptime = 90;
+        if (get_var('USE_SAP_HANA_SR_ANGI')) ? hanasr_angi_hadr_providers_setup($sid, $instance_id, $sapadm) : assert_script_run "su - $sapadm -c 'sapcontrol -nr $instance_id -function StartSystem HDB'";
+        my $looptime = 90;btrfs            
```

Comment:
```suggestion
        my $looptime = 90;
```

Reply:
Exactly, the funny part. DUNNO how it get there.